app/views/content/whats_new.en.html.md

```diff
@@ -6,13 +6,55 @@ ___
 
 <div style="padding: 10px; background-color: #E7F3FF; color: #333; margin-bottom: 15px;">
   <ul>
-    <li><b>November</b>: new production technologies will be added for the production of transport fuels and feedstock for the chemical industry, such as Fischer-Tropsch and pyrolysis. In addition, the use of transport fuels and feedstock for the chemical industry will be expanded for existing and newly added carriers (such as biomethanol and bionaphtha).</li>
-    <li><b>November</b>: PyETM will be formally released. It is a Python package to interact with the API of the ETM in a standardized way.</li>
+    <li><b>December</b>: it will be possible to set full load hours and upload custom profiles separately for the follwoing solar PV technologies: solar rooftop (households), solar rooftop (buildings), solar on land and solar offshore.</li>
   </ul>
 </div>
 
 ___
 
+## November 2025
+
+**Improved carbon modelling**
+
+The modelling of carbon-containing liquid fuels has been improved by adding new production processes and expanding final demand of the transport sector (national and international) and chemical industry.
+
+-> <img src="/assets/pages/whats_new/carbon_modelling_sankey_en.png" width="650" /> <-
+
+This improvement includes various new features and changes, among others:
+
+* A new section <a href="/scenario/supply/fuel_production/overview" target="_blank">Liquid fuels and feedstock</a> has been added, containing existing and new processes for liquid fuels production. Go to the <a href="https://docs.energytransitionmodel.com/main/liquid-fuels" target="_blank">documentation</a> for more information.
+* The following energy carriers have been added to the model: bionaphtha, biomethanol, pyrolysis oil and pyrolysis bio-oil.
+* Final demand in the <a href="/scenario/demand/industry/chemicals" target="_blank">chemical industry</a> has been expanded to include energetic demand for bionaphtha and non-energetic demand for bionaphtha, methanol and biomethanol.
+* Final demand in <a href="/scenario/demand/transport_freight_transport/domestic-navigation-technology" target="_blank">national</a> and <a href="/scenario/demand/transport_international_transport/international-navigation-technology" target="_blank">international</a> navigation has been expanded with demand for methanol and biomethanol. Additionally, final demand for biokerosene has been added to <a href="/scenario/demand/transport_international_transport/international-aviation-technology" target="_blank">national</a> aviation. The mix of these non-biogenic and biogenic fuels can be set in the <a href="/scenario/demand/transport_fuels/domestic-navigation" target="_blank">Transport fuels</a> section.
```

Comment:
I think this can be a bit shorter. I would just generally say that (inter)national transport and chemical industry have been expanded.